nestjs/src/courses/courseTasks/dto/course-task.dto.ts

```diff
@@ -73,4 +74,8 @@ export class CourseTaskDto {
   @IsNumber()
   @ApiProperty()
   scoreWeight: number;
+
+  @IsNotEmpty()
+  @ApiProperty()
+  crossCheckStatus: string;
```

Comment:
minor: you can define as enum in order to improve type-checking on ui. you can find example in our repo